src/Encoding.ts

```diff
@@ -1,31 +1,31 @@
-'use strict';
```

Comment:
tslint?   

One annoying thing about tslint is that it prefers double quote by default ...